distributed/cli/dask_worker.py

```diff
@@ -60,15 +60,15 @@
 @click.option('--interface', type=str, default=None,
               help="Network interface like 'eth0' or 'ib0'")
 @click.option('--nthreads', type=int, default=0,
-              help="Number of threads per process.")
+              help="Number of threads in the pool per process.")
```

Comment:
What does "in the pool" refer to here, the `ThreadPoolExecutor`?  If so I suggest that we drop this.  A novice user may not understand that threads come in pools or that dask uses a ThreadPool.  They just know that dask uses threads internally.

Reply:
That's right.